ArduCopter/system.cpp

```diff
@@ -206,11 +206,6 @@ void Copter::init_ardupilot()
 
     ins.set_log_raw_bit(MASK_LOG_IMU_RAW);
 
-    // enable output to motors
-    if (arming.rc_calibration_checks(true)) {
```

Comment:
Removing this might be fine but we need to be careful.  This was originally added because we would default to using the RC3_MIN/MAX for the motor range if MOT_PWM_MIN/MAX were not set.

The danger (which led to this code being added) was the user had a working vehicle and then did a parameter reset.  So for example this could happen:

1. user has a working vehicle with ESCs calibrated to use the 900 to 2100 pwm range
2. RC3_MIN/MAX set to 900 to 2100
3. vehicle is powered up at the time but disarmed
4. user resets their parameters so RC3_MIN = 1100, RC3_MIN = 1900
5. AP outputs "minimum" pwm value which is now 1100 and motors spin up

Reply:
> Removing this might be fine but we need to be careful. This was originally added because we would default to using the 
RC3_MIN/MAX for the motor range if MOT_PWM_MIN/MAX were not set.

Yep - but PH ensured those are set in 2021, so I think that's OK now.

I think this PR *may* lead to a change where the ESCs won't get signal until the vehicle is armed for the first time.  Which means they'll beep.  Same as what would happen if you booted with bad RC calibration right now.

We may want to consider under what other circumstances we don't want to output these minimum values.

Thinking about the scenario you outline - where parameters have been reset to ranges outside the ESC alibration ranges.  How are we saved on master in this scenario at the moment?  We don't require the MOT_PWM_MIN and MAX to be configured in storage, so the default values for those might be well inside the calibrated range of the ESC and thus cause the motors to spin.

> The danger (which led to this code being added) was the user had a working vehicle and then did a parameter reset. So for example this could happen:

Will run that example and see what happens.

I've created https://github.com/ArduPilot/ardupilot/pull/23314 to just remove the method to make this PR simpler.

Reply:
Yeah, I totally agree that the scenario I've outlined above could just as easily happen with MOT_PWM_MIN/MAX so I suspect we should always send the min output... e.g. remove the rc-calibrated check and just always call the output-min.